docs/changehistory/NextVersion.md

```diff
@@ -183,7 +183,15 @@ All non-internal components are deprecated with their corresponding replacements
 
 A number of **UI1.0** related APIs and components are deprecated and will be removed in the next `@itwin/appui-react` major version:
 `FrameworkVersion`, `FrameworkVersionContext`, `FrameworkVersionId`, `FrameworkVersionProps`, `ListPickerBase`, `useFrameworkVersion`,
-`NineZoneChangeHandler`, `StagePanelChangeHandler`, `WidgetStateFunc`, `ZoneDefProvider`.
+`NineZoneChangeHandler`, `StagePanelChangeHandler`, `WidgetStateFunc`, `ZoneDefProvider`, `Zone`, `ZoneDef`.
+
+Pseudo components used by the [FrontstageProvider]($appui-react) are deprecated and replaced by corresponding configuration interfaces:
+
+| Component    | Replacement                      |
+| ------------ | -------------------------------- |
+| `Frontstage` | [FrontstageConfig]($appui-react) |
+| `Widget`     | [WidgetConfig]($appui-react)     |
+| `StagePanel` | [StagePanelConfig]($appui-react) |
```

Comment:
arent `StagePanels` a `UI 1.0` concept? Completely replaced by widgets?
If so, do we eve need a replacement? Just mark as deprecated and completely remove in 4.0?

Reply:
We still have top/bottom/left/right panels in UI2.0 (where widgets can be docked). I was wondering if it would make sense to rename these to `PanelDef`, `PanelConfig` instead of using existing `StagePanelDef`..